zcash_client_sqlite/src/wallet/sapling.rs

```diff
@@ -235,6 +237,8 @@ pub(crate) fn select_spendable_sapling_notes(
                 AND sapling_received_notes.commitment_tree_position < unscanned.end_position_exclusive
                 -- exclude unscanned ranges above the anchor height which don't affect spendability
                 AND unscanned.block_range_start <= :anchor_height
+                -- exclude unscanned ranges that end below the wallet birthday
+                AND unscanned.block_range_end > :wallet_birthday
```

Comment:
11fc78c7e80f6d0d6bd83a2f684b9741e86de26e does not currently run, because these two lines should be part of 1bd64b2c4afeb1acc9131627ba94f378a3671041.